include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -497,8 +489,9 @@ struct GemmKernel
         // Run Epilogue Pipeline
         auto& c_block_window = gemm_tile_windows.at(I2);
 
-        if constexpr(DstInMemOp == memory_operation_enum::set || sizeof(CDataType) > 2 ||
-                     EpiloguePipeline::GetVectorSizeC() % 2 == 0)
+        if constexpr(DstInMemOp == memory_operation_enum::set ||
+                     !(EpiloguePipeline::GetVectorSizeC() % 2 != 0 &&
+                       is_any_of<CDataType, fp16_t, bf16_t>::value))
```

Comment:
I wonder right now is there a way that we won't even let user to run such code. If this won't compile then I thinks that's fine. At least that is a very clear message that such configuration won't work. Because right now those condition may let sb think that there are cases when epilogue is not run at all.